Doc/whatsnew/3.14.rst

```diff
@@ -417,6 +417,16 @@ New Features
   which has an ambiguous return value.
   (Contributed by Irit Katriel and Erlend Aasland in :gh:`105201`.)
 
+* :c:func:`Py_Finalize` now deletes all interned strings. This
+  is backwards incompatible to any C-Extension that holds onto an interned
+  string after a call to :c:func:`Py_Finalize` and is then reused after a
+  call to :c:func:`Py_Initialize`.  Any issues arising from this behavior will
+  normally result in crashes during the exectuion of the subsequent call to
+  :c:func:`Py_Initialize` from accessing uninitialized memory. To fix, use
+  an address sanitizer (i.e. ASAN) to identify any use-after-free coming from
```

Comment:
I don't think the acronym is useful here. It's not used again, and I don't think searching for “ASAN” will get you much more than “address sanitizer”.
```suggestion
  an address sanitizer to identify any use-after-free coming from
```
If you want to keep it, [we prefer](https://devguide.python.org/documentation/style-guide/#use-simple-language) English terms (“that is”) over Latin abbreviation like “i.e.”.